README.md

```diff
@@ -49,8 +49,7 @@ a more holistic understanding of the various Link options.
 <PlaidLink
   clientName="Your app name"
   env="sandbox"
-  institution={null}
-  publicKey={PLAID_PUBLIC_KEY}
+  key={PLAID_PUBLIC_KEY}
```

Comment:
We also need to modify these spots to do `s/publicKey/key`: https://github.com/plaid/react-plaid-link/search?q=publicKey&unscoped_q=publicKey

Reply:
https://github.com/plaid/react-plaid-link/pull/72

Reply:
Good catch – I got mixed up by the `Plaid.create` call. Better to have a 1:1 mapping from the React props to the Link configuration parameter names in our docs.